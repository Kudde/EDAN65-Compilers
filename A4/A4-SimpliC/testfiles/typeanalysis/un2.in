int fun() {
  int k = 3 == 1;
  int y = 3;
  if (true < y) {
        y = 4;
  }
}
