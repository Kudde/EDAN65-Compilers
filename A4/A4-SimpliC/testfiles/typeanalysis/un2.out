Error at line 4: type unknownType should be boolType
