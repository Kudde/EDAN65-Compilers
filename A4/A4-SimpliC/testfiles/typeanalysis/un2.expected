Error at line 3: Incompatible types!
